nri/config.go

```diff
@@ -0,0 +1,54 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package nri
+
+import (
+	nri "github.com/containerd/nri/v2alpha1/pkg/adaptation"
+)
+
+// Config data for NRI.
+type Config struct {
+	Disable    bool   `toml:"disable" json:"disable"`
```

Comment:
pls add descriptions for config fields.. defaults etc.. esp. since disable default is true for now..

Reply:
need a docs/nri.md file, and/or edits in plugins.md, at least to describe purpose, setup, these config fields.. and provide links to other relevant docs.. bonus points for a diagram

Reply:
I added a docs/NRI.md. It briefly describes what NRI is, how the containerd/NRI integration is set up and why. There is a pointer to the NRI repo for further information (although that is not very useful before the related pending NRI(v2) PR gets merged there).